surveys.yml

```
@@ -160,37 +160,37 @@ translations:
   - key: general.faq
     t: FAQ
   - key: faq.create_account
-    t: Why do I need to create an account?
+    t: 계정을 생성해야 하는 이유는 무엇인가요?
   - key: faq.create_account.description
-    t: We ask you to create an account in order to avoid duplicate responses, save your data, and notify you when results are published.
+    t: 중복 응답을 방지하고, 데이터를 저장하며, 결과가 게시될 때 알려드리기 위해 계정을 만들어 주시기 바랍니다.
   - key: faq.anonymous_survey
-    t: Can I take the survey anonymously?
+    t: 익명으로 설문조사에 참여할 수 있나요?
   - key: faq.anonymous_survey.description
-    t: Yes, you can use a fake (something@example.com) or non-identifying email to take the survey as long as you save it for future access.
+    t: 예, 불특정 이메일 (something@example.com) 또는 식별되지 않는 이메일을 사용하여 설문조사에 참여할 수 있습니다.
   - key: faq.questions_required
-    t: Is answering every question required?
+    t: 모든 질문에 반드시 답변해야 하나요?
   - key: faq.questions_required.description
-    t: No, every question of the survey is optional and can freely be skipped.
+    t: 아니요, 설문조사의 모든 질문은 선택 사항이며 자유롭게 건너뛸 수 있습니다.
   - key: faq.data_published
-    t: Will my data be made public?
+    t: 제 데이터는 공개되나요?
   - key: faq.data_published.description
-    t: Yes, all data will be released publicly, but only after being purged of any identifying information (emails, usernames, IDs, etc.).
+    t: 예, 모든 데이터는 공개적으로 공개되지만 개인 정보(이메일, 사용자 아이디, 사용자 이름 등)가 제거된 후에만 공개됩니다.
   - key: faq.survey_design
-    t: How are the surveys designed?
+    t: 설문조사는 어떤 식으로 설계되어 있나요?
   - key: faq.survey_design.description
-    t: Surveys are designed based on community feedback, as well as reviewed by experts. You can [learn more here](https://dev.to/sachagreif/how-the-state-of-js-css-surveys-are-run-4lnb).
+    t: 설문조사는 커뮤니티의 피드백을 바탕으로 설계되고 전문가들의 검토를 거칩니다. 자세한 내용은 여기에서 확인할 수 있습니다(https://dev.to/sachagreif/how-the-state-of-js-css-surveys-are-run-4lnb).
```

링크를 유지할 수 있으면 좋겠습니다.
```suggestion
    t: 설문조사는 커뮤니티의 피드백을 바탕으로 설계되고 전문가들의 검토를 거칩니다. [자세한 내용은 여기에서 확인할 수 있습니다](https://dev.to/sachagreif/how-the-state-of-js-css-surveys-are-run-4lnb).
```